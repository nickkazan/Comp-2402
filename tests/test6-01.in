Hello
Nick
Dog
Buddy
Branch
Cat
