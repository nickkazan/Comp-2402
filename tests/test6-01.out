Cat
Dog
Nick
Buddy
Hello
Branch
